synthtool/gcp/gapic_microgenerator.py

```diff
@@ -128,22 +131,28 @@ def _generate_code(
 
         # The time has come, the walrus said, to talk of actually running
         # the code generator.
-        log.debug(f"Generating code for: {proto_path}.")
         sep = os.path.sep
-        shell.run(
-            [
-                "docker",
-                "run",
-                "--mount",
-                f"type=bind,source={googleapis / proto_path}{sep},destination={Path('/in') / proto_path}{sep},readonly",
-                "--mount",
-                f"type=bind,source={output_dir}{sep},destination={Path('/out')}{sep}",
-                "--rm",
-                "--user",
-                str(os.getuid()),
-                f"gcr.io/gapic-images/gapic-generator-{language}",
-            ]
-        )
+        docker_run_args = [
+            "docker",
+            "run",
+            "--mount",
+            f"type=bind,source={googleapis / proto_path}{sep},destination={Path('/in') / proto_path}{sep},readonly",
+            "--mount",
+            f"type=bind,source={output_dir}{sep},destination={Path('/out')}{sep}",
+            "--rm",
+            "--user",
+            str(os.getuid()),
+            f"gcr.io/gapic-images/gapic-generator-{language}",
```

Comment:
nit: let's add `:{generator_version}` here, otherwise it's never used  (e.g. `:latest`)